.vscode/settings.json

```diff
@@ -6,6 +6,7 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
   },
+  "typescript.preferences.autoImportFileExcludePatterns": ["index.ts"],
```

Comment:
This will only affect _this_ repo, right? Is this a fix for the bug where auto-import would import files from `src/...` instead of `<package_name>`? If yes, are we sure that this issue was limited to our repo only?

Reply:
Yes, this setting affects only this repo — it ignores `index.ts` files when suggesting auto-import paths, reducing circular-dependency risks. These `index.ts` files are the package's entry point for the external world, but internally, all imports should point to their destination file directly.

The fix for the `src/` was done by removing `baseUrl` from `tsconfig.json`.

Reply:
That makes sense, thanks @arboleya

Reply:
It turns out this didn't work as planned.

The `index.ts` files are ignored even from outside the package.

So imports like this:

```ts
import {
  getSupportedVersions,
  getUserVersions,
  compareUserVersions,
  colorizeUserVersion,
} from '@fuel-ts/versions';
```

Would be suggested like this:

```ts
import { colorizeUserVersion } from '@fuel-ts/versions/src/lib/colorizeUserVersion';
import { compareUserVersions } from '@fuel-ts/versions/src/lib/compareUserVersions';
import { getSupportedVersions } from '@fuel-ts/versions/src/lib/getSupportedVersions';
import { getUserVersions } from '@fuel-ts/versions/src/lib/getUserVersions';
```

I wonder why it wasn't suggesting it like this on my previous tests.

Anyway, I'm already on it.

Reply:
Removed on:
 - https://github.com/FuelLabs/fuels-ts/pull/902